src/audio/lib/thread.c

```diff
@@ -296,24 +296,24 @@ void AudioThread_ProcessGlobalCmd(AudioCmd* cmd) {
             break;
 
         case AUDIOCMD_OP_GLOBAL_SET_CUSTOM_UPDATE_FUNCTION:
-            gAudioCustomUpdateFunction = cmd->asUInt;
+            gAudioCustomUpdateFunction = cmd->asPtr;
             break;
 
         case AUDIOCMD_OP_GLOBAL_SET_CUSTOM_FUNCTION:
             if (cmd->arg2 == AUDIO_CUSTOM_FUNCTION_REVERB) {
-                gAudioCustomReverbFunction = cmd->asUInt;
+                gAudioCustomReverbFunction = cmd->asPtr;
             } else if (cmd->arg2 == AUDIO_CUSTOM_FUNCTION_SYNTH) {
-                gAudioCustomSynthFunction = cmd->asUInt;
+                gAudioCustomSynthFunction = cmd->asPtr;
             } else {
-                gAudioCtx.customSeqFunctions[cmd->arg2] = cmd->asUInt;
+                gAudioCtx.customSeqFunctions[cmd->arg2] = cmd->asPtr;
             }
             break;
 
         case AUDIOCMD_OP_GLOBAL_SET_DRUM_FONT:
         case AUDIOCMD_OP_GLOBAL_SET_SFX_FONT:
         case AUDIOCMD_OP_GLOBAL_SET_INSTRUMENT_FONT:
             if (AudioPlayback_SetFontInstrument(cmd->op - AUDIOCMD_OP_GLOBAL_SET_DRUM_FONT, cmd->arg1, cmd->arg2,
-                                                cmd->asInt)) {}
+                                                (void*)cmd->asUInt)) {}
```

Comment:
Should this use `asPtr`?

Reply:
Yeah, this can be `asPtr` assuming it doesn't cause any issues